packages/cli/src/cmds/preview/preview.ts

```diff
@@ -452,15 +458,10 @@ export default class Preview extends YargsCommand {
       return err(configResult.error);
     }
     const config = configResult.value;
-    tenantId = config?.config
-      ?.get(constants.solutionPluginName)
-      ?.get(constants.teamsAppTenantIdConfigKey) as string;
-    localTeamsAppId = config?.config
-      ?.get(constants.appstudioPluginName)
-      ?.get(constants.remoteTeamsAppIdConfigKey);
-    localBotId = config?.config
-      ?.get(constants.botPluginName)
-      ?.get(constants.botIdConfigKey) as string;
+    tenantId = config?.config?.[constants.appstudioPluginName]?.["tenantId"] as string;
```

Comment:
Just double confirm - now tenantId is part of appstudio?

Reply:
yes